modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/render.tsx

```diff
@@ -14,12 +14,11 @@
 
 import {Provider} from '@clayui/core';
 import {ClayIconSpriteContext} from '@clayui/icon';
+import {CONSTANTS, accessibilityMenuAtom} from '@liferay/accessibility-settings-state-web';
```

Comment:
Do the destructured variables need to be sorted? Is that why SF is failing?

Reply:
@ethib137 maybe it is, but when I change the order it causes an SF error too, I think it's related to the number of columns in the line that has 92 the maximum is 80.